pkg/slurm/Create.go

```diff
@@ -166,32 +141,21 @@ func (h *SidecarHandler) SubmitHandler(w http.ResponseWriter, r *http.Request) {
 
 		// prepareEnvs creates a file in the working directory, that must exist. This is created at prepareMounts.
 		envs := prepareEnvs(spanCtx, h.Config, data, container)
+		image = prepareImage(spanCtx, h.Config, metadata, container.Image)
+		commstr1 := prepareRuntimeCommand(h.Config, container, metadata)
+		log.G(h.Ctx).Debug("-- Appending all commands together...")
+		runtime_command := append(commstr1, envs...)
+		switch h.Config.ContainerRuntime {
```

Comment:
It's ok defaulting to singularity, I wonder if maybe we can rise an fatal error, earlier in the code, if the runtime name is not singularity neither enroot. 

It can happen that one user blindly put in the config a runtime that is not in that list.